### PR TITLE
Fix for LinkWithMesh causes some flickering on button creation

### PR DESCRIPTION
**Pull request by msDestiny14**
https://github.com/BabylonJS/Babylon.js/issues/11014

### CODE REVIEW

**Review by sebavan — COMMENTED** [review]
Looks way better, just need to do it for the line component as well :-)

Only question is when could parent._currentMeasure be unavailable ? to be sure it does not flicker in most cases ?

**Comment by msDestiny14** [comment]
I'm trying to think of a case. Parent should always be available when that function is called because it will be inside the adt so _root will always be the parent worst case.

**Comment by sebavan** [comment]
Yup, I was more focused on the _currentMeasure  of the parent :-)

File: gui/src/2D/controls/control.ts [hunk 1]
```diff
@@ -1318,6 +1318,13 @@ export class Control {
         let oldLeft = this._left.getValue(this._host);
         let oldTop = this._top.getValue(this._host);
 
+        if (this._currentMeasure.width == 0 && this._currentMeasure.height == 0) {
```
Comment (RaananW) [inline gui/src/2D/controls/control.ts:1321]:
```suggestion
        if (this._currentMeasure.width === 0 && this._currentMeasure.height === 0) {
```

**Review by RaananW — APPROVED** [review]

**Pull request merged**